.github/workflows/security.yml

```
@@ -0,0 +1,379 @@
+        uses: tenable/terrascan-action@main
+        with:
+          iac_type: 'docker'
+          iac_dir: '.'
+          sarif_upload: true
+        continue-on-error: true
+
+  # Container scanning (if Dockerfile exists)
+  container-scan:
+    name: Container Security Scan
+    runs-on: ubuntu-latest
+    if: hashFiles('.devcontainer/Dockerfile') != ''
```

actionlint (1.7.7)

calling function "hashFiles" is not allowed here. "hashFiles" is only available in "jobs.<job_id>.steps.continue-on-error", "jobs.<job_id>.steps.env", "jobs.<job_id>.steps.if", "jobs.<job_id>.steps.name", "jobs.<job_id>.steps.run", "jobs.<job_id>.steps.timeout-minutes", "jobs.<job_id>.steps.with", "jobs.<job_id>.steps.working-directory". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)